commands/intakecommands.py

```diff
@@ -0,0 +1,89 @@
+from commands2 import ParallelCommandGroup
+from commands.elevatorsetting import (
+    ElevatorBottomPosition,
+    ElevatorAmpPosition,
+    ElevatorTopPosition,
+)
+from commands.intakesetting import (
+    FloorIntake,
+    StageIntake,
+    FeedIntakeToShooter,
+    EjectInAmp,
+    HoldIntakeAtHandoff,
+    EjectInTrap,
+)
+from subsystems.elevatorsubsystem import ElevatorSubsystem
+from subsystems.intakesubsystem import IntakeSubsystem
+from subsystems.shootersubsystem import ShooterSubsystem
+
+
+class GroundIntake(ParallelCommandGroup):
+    def __init__(self, elevator: ElevatorSubsystem, intake: IntakeSubsystem):
+        ParallelCommandGroup.__init__(
+            ElevatorBottomPosition(elevator), FloorIntake(intake)
+        )
+        self.setName(__class__.__name__)
+
+
+class DefaultIntake(ParallelCommandGroup):
+    def __init__(self, elevator: ElevatorSubsystem, intake: IntakeSubsystem):
+        ParallelCommandGroup.__init__(
+            ElevatorBottomPosition(elevator), HoldIntakeAtHandoff(intake)
+        )
+        self.setName(__class__.__name__)
+
+
+class PrepareAmp(ParallelCommandGroup):
+    def __init__(self, elevator: ElevatorSubsystem, intake: IntakeSubsystem):
+        ParallelCommandGroup.__init__(
+            ElevatorAmpPosition(elevator), StageIntake(intake)
+        )
+        self.setName(__class__.__name__)
+
+
+class PrepareTrap(ParallelCommandGroup):
+    def __init__(self, elevator: ElevatorSubsystem, intake: IntakeSubsystem):
```

Comment:
because of the chain the sequencing might want to be such that it first stages the intake, and then elevatortopposition